### ISSUE TITLE
How to add left and right view in TabBar?

### ISSUE THREAD

**Issue by musicode** [react-native-community/react-native-tab-view#593]
![image](https://user-images.githubusercontent.com/2732303/44647558-edc35400-aa10-11e8-8c5e-abe513c8cf19.png)

In the screenshot, I want to add a search button to the left of TabBar and a more buutton to the right of TabBar.

at the same time, I hope to set the tab width, so that the underline should be more shorter.

**Comment by ibmsoft** [comment]
@musicode  are you resolve this problem

**Comment by musicode** [comment]
@ibmsoft don't use the TabBar, I write my own TabBar like below:

```
<View>
  {left}
  <ScrollView></ScrollView>
  {right}
</View>
```

**Comment by Volekss** [comment]
> @ibmsoft don't use the TabBar, I write my own TabBar like below:
> 
> ```
> <View>
>   {left}
>   <ScrollView></ScrollView>
>   {right}
> </View>
> ```

can you share you code for this approach?

**Comment by satya164** [comment]
Hey, I just released a new alpha `2.0.0-alpha.0` of the library. It's rewritten using [react-native-gesture-handler](https://github.com/kmagiera/react-native-gesture-handler) and [react-native-reanimated](https://github.com/kmagiera/react-native-reanimated) addresses a many platform specific bugs and performance problems. The documentation is updated as well.

Please try the new version and see if it addresses your issue. If not, please open a new issue following the issue template.
